package.json

```diff
@@ -17,11 +17,11 @@
   },
   "homepage": "https://github.com/complate/complate-fractal",
   "dependencies": {
-    "@babel/core": "~7.20.12",
-    "@babel/plugin-transform-react-jsx": "~7.20.13",
-    "@babel/preset-env": "~7.20.2",
-    "@babel/register": "~7.18.9",
-    "fractal-fork": "^0.1.0",
+    "@babel/core": "~7.23.7",
+    "@babel/plugin-transform-react-jsx": "~7.23.4",
+    "@babel/preset-env": "~7.23.7",
```

Comment:
FWIW: [npm-check-updates](https://www.npmjs.com/package/npm-check-updates) reveals that there's a v7.23.8 already - not sure whether you wanna update that while you're at it

There are also updated versions for Prettier and ESLint, but I assume those were omitted intentionally?

Reply:
I will update babel again.

I omitted Prettier and ESLint intentionally.